rovercode/app.py

```diff
@@ -69,6 +74,20 @@ def on_message(_, raw_message):
         MOTOR_CONTROLLER.set_speed(message[constants.MOTOR_ID_FIELD],
                                    message[constants.MOTOR_VALUE_FIELD],
                                    message[constants.MOTOR_DIRECTION_FIELD])
+    elif message_type == constants.CHAINABLE_RGB_LED_COMMAND:
+        led_id, red_value, green_value, blue_value = \
+            message.get(constants.CHAINABLE_RGB_LED_ID_FIELD), \
+            message.get(constants.CHAINABLE_RGB_LED_RED_VALUE_FIELD), \
+            message.get(constants.CHAINABLE_RGB_LED_GREEN_VALUE_FIELD), \
+            message.get(constants.CHAINABLE_RGB_LED_BLUE_VALUE_FIELD)
+        if led_id > CHAINABLE_RGB_MANAGER.count - 1:
```

Comment:
This can be another `range` check

Reply:
Changing this to a null check -- no reason to have the same range checking logic at every level!

Reply:
Actually, the lower level range checks will catch None, too. So let's remove these checks altogether and let the lower level do input verification. This top-level parser is too crowded anyway.